graphql/data/schema.js

```diff
@@ -746,6 +747,16 @@ const userType = new GraphQLObjectType({
       description: 'feature values for this specific user',
       resolve: (user, args, context) => getUserFeatures(context, user),
     },
+    searchEngine: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'the User’s search engine',
+      resolve: user => getUserSearchEngine(user),
+    },
+    showYahooPrompt: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'whether to show the yahoo search prompt',
+      resolve: user => !user.yahooSearchSwitchPrompt.hasSeenPrompt,
```

Comment:
This should depend on the user's experiment group, right? Check experiment group; if the user's in the control or not assigned, false; else if in treatment,  true if they haven't already seen the prompt (false if they have already seen it).

Reply:
Yeah that's fair. I was taking the ticket pretty literally and thinking we could combine it with the bucket in the front-end, but probably better to do the calculation in one place if we're able to do so.